tests/osgUtil/LineSegmentIntersector.js

```diff
@@ -23,37 +23,37 @@ module.exports = function() {
         // start right on the edge
```

Comment:
I'm missing unit testing for the new functionality.

Reply:
yep needs to add test for line/point lineSegmentIntersector and line/point for SphereIntersector.